annoyed-alligators/socl_media/apps/feed/views.py

```diff
@@ -7,6 +7,12 @@
 
 @login_required
 def feed(request):
```

Comment:
For this, there's a built in Django ListView. It's a class based view and does most of the work internally. Please use the built-in view. 
Here's how to use them: https://youtu.be/Xeh9r0CXBmU